packages/react-native-bpk-component-phone-input/src/common-types.js

```diff
@@ -62,6 +66,7 @@ export const LIST_COMMON_PROP_TYPES = {
 
 export const LIST_COMMON_DEFAULT_PROPS = {
   selectedId: null,
+  suggested: undefined,
```

Comment:
Just to be consistent with other default prop values:

```suggestion
  suggested: null,
```